lib/oli_web/controllers/project_controller.ex

```diff
@@ -173,4 +174,18 @@ defmodule OliWeb.ProjectController do
     )
   end
 
+  def clone_project(conn, _project_params) do
+    case Clone.clone_project(conn.assigns.project.slug, conn.assigns.current_author) do
+      {:ok, project} ->
+        IO.inspect(project)
+        conn
+        |> put_flash(:info, "Project copied. You've been redirected to your new project.")
```

Comment:
The language here should probably be consistent with the operation name on the button that was clicked. For example, "Project duplicated. You are now viewing your new project.". Project copied terminology might indicate it has been copied to the clipboard or something and further steps are required